packages/razzle-start-server-webpack-plugin/src/StartServerPlugin.js

```diff
@@ -118,26 +118,22 @@ export default class StartServerPlugin {
   }
 
   _handleChildExit(code, signal) {
-    if (code) this._error('script exited with code', code);
-
+    this._error(`Script exited with ${signal ? `signal ${signal}` : `code ${code}`}`);
     this.worker = null;
 
-    if (signal && signal !== 'SIGTERM'){
-      this._error('script exited after signal', signal);
-      process.exit()
-      return;
-    }
-    if (!this.workerLoaded) {
```

Comment:
Handle this in error handler aswell?

Reply:
Do you mean provide a way to also call process.exit() in the child process error handler instead of just setting this.worker to null? (and then safeguard for the case where both exit and error are triggered)